### DISCUSSION TITLE
JupyterCon 2023 JupyterLite sprint?

### DISCUSSION THREAD

**Discussion by tylere** [jupyterlite/jupyterlite#1057] [General]
Is there going to be a JupyterLite [sprint](https://www.jupytercon.com/sprints) at JupyterCon 2023?

**Comment by jtpio** [comment]
It would be fun to have one yes, thanks for asking.

I would have to leave on Sunday and can only do one day though.

**Comment by tylere** [comment]
The JupyterCon signup form for those "interested in leading a sprint" is now live: [Sprints](https://www.jupytercon.com/sprints)

@jtpio would you be willing to be the "Sprint Leader"?

**Reply by jtpio** [reply]
Sure, but I can only be there on Saturday 13.

Is there any topics you would like to cover in particular? Third-party kernels, extensions, contents management?
Otherwise the sprints could also be a good time to address some issues for the 0.2.0 release.

**Comment by tylere** [comment]
My main interest is in getting more hands-on time with JupyterLite in the company of other developers/users, in order to better understand its capabilities and pain points.

In terms of functionality, I am interested in making the process for adding non-standard packages clearer. One idea would be to create a How-To Guide describing how to take an existing Binder and/or Colab example and convert it to JupyterLite. Describe the pros/cons of installing packages at runtime vs. bundling in Pyodide, etc.

I'm particularly interested in geospatial packages, and they often have non-Python dependencies.

**Reply by jtpio** [reply]
Thanks for the suggestions! Yes having some time to discuss these points and document them in the docs would be great :+1:

**Comment by tylere** [comment]
@jtpio will you fill out the signup form for registering the sprint, or would you like me to do so?

**Reply by jtpio** [reply]
Please feel free to fill out the signup form, thanks!

**Comment by tylere** [comment]
I'm happy to take the lead on any logistics that arise, if you will be the technical lead for the sprint on Saturday.

I submitted the following information:
- **Email Address:** tylerickson@gmail.com
- **Name of Sprint Leader:** Jeremy Tuloup (QuantStack)
- **Sprint name or theme:** JupyterLite (Jupyter running in the browser)
- **Minimum level of expertise needed:** Intermediate
- **Is familiarity with package required?** No
- **Github repository link:** https://github.com/jupyterlite/jupyterlite
- **Contributor guidelines link:** https://jupyterlite.readthedocs.io/en/latest/contributing.html
- **Description of the goal(s) of the sprint, tasks planned an why it is important:** JupyterLite is a JupyterLab distribution that runs entirely in the browser, and has the potential to dramatically increase access to Jupyter by reducing the reliance on maintaining host servers (JupyterHub, Binder, etc.). This sprint will focus on improving documentation and addressing issues for the 0.2.0 release.

Let me know if you have any suggestions for changes... I can edit the form submission.

**Reply by jtpio** [reply]
Thanks!

Yes I'm happy to be the technical lead on Saturday. Feel free to continue the sprint on Sunday as well, maybe some other people would be interested.
